IPlug/IPlugProcessor.cpp

```diff
@@ -509,11 +512,16 @@ void IPlugProcessor::PassThroughBuffers(PLUG_SAMPLE_SRC type, int nFrames)
 
 void IPlugProcessor::ProcessBuffers(PLUG_SAMPLE_DST type, int nFrames)
 {
+  mMeterLevelIn = GetInputBufferMaxValue(nFrames);
```

Comment:
This is going to affect all plug-in formats. This should move out of IPlugProcessor